LicenseHeaderManager/PackageCommands/RemoveHeaderFromAllFilesInSolutionCommand.cs

```diff
@@ -20,14 +20,14 @@
 
 namespace LicenseHeaderManager.PackageCommands
 {
-  class RemoveLicenseHeaderFromAllProjectsCommand : ISolutionLevelCommand
+  public class RemoveHeaderFromAllFilesInSolutionCommand : ISolutionLevelCommand
   {
     private const string c_commandName = "Remove LicenseHeader from all Projects";
 
     private readonly IVsStatusbar _statusBar;
     private readonly LicenseHeaderReplacer _licenseReplacer;
 
-    public RemoveLicenseHeaderFromAllProjectsCommand(IVsStatusbar statusBar, LicenseHeaderReplacer licenseReplacer)
+    public RemoveHeaderFromAllFilesInSolutionCommand(IVsStatusbar statusBar, LicenseHeaderReplacer licenseReplacer)
     {
       this._statusBar = statusBar;
       this._licenseReplacer = licenseReplacer;
```

Comment:
Again, mistake by me... i am going to clean this up myself after the pull request